internal/api/server.go

```diff
@@ -15,21 +15,24 @@ import (
 )
 
 type ConfigYAML struct {
-	KeyPath  string
-	CertPath string
-	DBPath   string
-	Port     int
+	KeyPath     string
+	CertPath    string
+	DBPath      string
+	FrontendDir string
```

Comment:
I've made it possible to provide any directory for the frontend, but I'm not sure if this is a good idea.